lib/cache/cache.go

```diff
@@ -152,6 +153,7 @@ func ForProxy(cfg Config) Config {
 		{Kind: types.KindInstaller},
 		{Kind: types.KindKubernetesCluster},
 		{Kind: types.KindSAMLIdPServiceProvider},
+		{Kind: types.KindUserGroup},
```

Comment:
Keep in mind that new resources can only be added to non-auth caches during major versions, meaning this cannot be backported to any existing release.

We're working on ways to change this, but for now, adding a resource in a peripheral cache during a minor/patch update breaks our compatibility guarantees.